lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -62,6 +62,12 @@ def user
     account
   end
 
+  # TODO: this has to be supplied to still use the old provider for managing tasks
+  # technically needed as a bug fix
+  def compatibility
```

Comment:
Need to compare to original commit that added compatibility to make sure everything is added as necessary here.